container-toolkit/index.md

```diff
@@ -23,16 +23,19 @@ Container Device Interface <cdi-support.md>
 docker-specialized.md
 ```
 
-The NVIDIA Container Toolkit enables users to build and run GPU-accelerated containers.
-The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container)
-and utilities to automatically configure containers to leverage NVIDIA GPUs.
+The NVIDIA Container Toolkit is a collection of libraries and utilities enabling users to build and run GPU-accelerated containers.
```

Comment:
- Removed the "includes ..." part of the sentence, because of the new complete enumeration just below.
- I do like the sentence "... to automatically configure containers to leverage NVIDIA GPUs." and I also like "...to build and run GPU-accelerated containers". I think in the overview we do not need both. What we end up with clearly is a matter of taste. I am just proposing some kind of simplification.